ideascube/conf/kb_rwa_giz.py

```diff
@@ -0,0 +1,13 @@
+"""KoomBook conf"""
+from .kb import *  # noqa
+
+LANGUAGE_CODE = 'en'
+IDEASCUBE_NAME = 'GIZ'
```

Comment:
I'm using set server name currently in the playbook, however the name picked up to set the var is the project name which is not really nice...I'll remove IDEASCUBE_NAME

People will have to find their way to the admin page to change the displayed name...